ext/POSIX/POSIX.xs

```diff
@@ -13,6 +13,23 @@
 
 static int not_here(const char *s);
 
+#if defined(__MINGW32__) && !defined(USE_QUADMATH)
+/* If nvtype is long double, the bessel functions still *
+ * operate at "double precision" only - as in the past. *
```

Comment:
Why trailing `*` in these two lines?

Reply:
probably word-wrapped it after some editing, I'll fix it

Reply:
Fixed.